gebru/roster.json

```diff
@@ -154,6 +154,10 @@
       "materials": [
         {
           "type": "cv"
+        },
+        {
+          "type": "video",
+          "url": "<https://youtu.be/8wUbYOIJvHA>"
```

Comment:
@jianli1028
2 things to mention
1. I know the name of the key is misleading, but can you please replace the full url with the video_id i.e, "url": "pSXnMD5ug7Q"
2. The url that you typed, would not have worked because of the angle brackets, so be careful :)

Reply:
yeah my bad, just pushed the correct one